feeds/views.py

```
@@ -1 +1,66 @@
+    pagination_class = FeedPagination
+
+    def get_queryset(self):
+        user = self.request.user
+        sort = self.request.query_params.get("sort")
+        followed_user_ids = Followers.objects.filter(
+            follower=user,
+            unfollowed_at__isnull=True
+        ).values_list("following_id", flat=True)
+        list_of_ids = list(followed_user_ids) + [user.id]
+
+        #make table to mute and block users
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)